dist/tools/vera++/profiles/riot

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/tclsh
+
+set rules {
+  L004
```

Comment:
this is also in the force profile, to me it seems like either this or the other profile but not both, right?

Reply:
ah sorry, just saw that this goes with the parameter below, so please ignore